MekHQ/src/mekhq/campaign/parts/MissingAeroSensor.java

```diff
@@ -57,7 +57,12 @@ public MissingAeroSensor(int tonnage, boolean drop, Campaign c) {
     
     @Override 
 	public int getBaseTime() {
-		return 1200;
+        //Published errata for replacement times of small aero vs large craft
+        Entity e = unit.getEntity();
+        if (e.hasETypeFlag(Entity.ETYPE_DROPSHIP) || e.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+            return 1200;
+        }
```

Comment:
This is an example of what I meant in the comment above, 1200 for large space vessels is ~4.62* the small craft value. Not consistent at all. We should be proposing a more consistent and simplified errata I think :/